### DISCUSSION TITLE
How to improve camera stream quality when scanning for marker ?

### DISCUSSION THREAD

**Discussion by udayk07** [sceneview/sceneform-android#402] [Q&A]
We are trying to implement marker scan feature in AR. While trying to scan marker, camera stream almost blurry ( as in screenshot).

Tried to update CameraConfig in Session object with highest cpuImageSize & gpuTextueSize ( from received configs session.getSupportedCameraConfigs(cameraConfigFilter) ) , but it is not working for us.

Could anyone help us to improve the camera quality ?
Thanks.

![ar_marker_scan_camera_blur](https://user-images.githubusercontent.com/78372917/183675351-625531cc-d148-450c-8e70-70c2f665ac87.jpg)

**Comment by ThomasGorisse** [comment]
Hi.
It's an ARCore issue.
You have to set the config auto focus mode after the session creation.
E.g. after `onResume()`
Or move to SceneView.

**Reply by udayk07** [reply]
@ThomasGorisse Thanks for the info. Tried to pause ( arSceneView.pause() ) and resume ( arSceneView.resume() ) the session, it works fine.
